shotover-proxy/tests/runner/hotreload_int_tests.rs

```diff
@@ -0,0 +1,10 @@
+use crate::shotover_process;
+#[tokio::test]
+async fn test_hotreload_cli_flag_accepted() {
+    let shotover_process = shotover_process("tests/test-configs/null-valkey/topology.yaml")
+        .with_hotreload(true) // Use the new method we'll create
```

Comment:
`// Use the new method we'll create` this comment only makes sense in the context of when its initially written or reviewed. If I read this code in 5 months, the comment wont make any sense, so lets remove it.